server/middleware/setUpCsrf.ts

```diff
@@ -14,7 +14,7 @@ export default function setUpCsrf(): Router {
       // By default, csrf-sync uses x-csrf-token header, but we use the token in forms and send it in the request body, so change getTokenFromRequest so it grabs from there
       getTokenFromRequest: req => {
         // eslint-disable-next-line no-underscore-dangle
-        return req.body._csrf || req.query._csrf
+        return req.body?._csrf || req.query?._csrf
```

Comment:
req.body returns undefined now (was {} before) and so this errors out - adding a ? will ensure it gets to the alternative
https://expressjs.com/en/guide/migrating-5.html#req.body